app/controllers/borrows_controller.rb

```diff
@@ -7,6 +7,20 @@ def show
     @borrow = Borrow.find(params[:id])
   end
 
+  def new
+    @borrow = Borrow.new
+  end
+
+  def create
+    @borrow = Borrow.new(borrow_params)
```

Comment:
currently `borrow_params` allow only `user_id`, lets extend it with all needed params